app/models/setting.rb

```diff
@@ -17,6 +17,8 @@
 
 # RailsSettings Model
 class Setting < ApplicationRecord
+
+  SEPARATOR_REGEXP = /[\n,;]+/.freeze
```

Comment:
This is an related bugfix to deal with a type of setting that can, but doesn't yet, exist